### ISSUE TITLE
Unit test breaks in master

### ISSUE THREAD

**Issue by pmorie** [openshift/open-service-broker-sdk#6]
I get the following:

```
  github.com/openshift/open-service-broker-sdk/pkg/apis/broker/install
--- FAIL: TestRoundTripTypes (0.00s)
        roundtrip.go:111: starting group "sdkbroker.broker.k8s.io"
        roundtrip.go:141: round tripping sdkbroker.broker.k8s.io/__internal, Kind=ServiceInstanceList
        roundtrip.go:199:       round tripping to sdkbroker.broker.k8s.io/v1alpha1, Kind=ServiceInstanceList v1alpha1.ServiceInstanceList
        roundtrip.go:199:       round tripping to sdkbroker.broker.k8s.io/v1alpha1, Kind=ServiceInstanceList v1alpha1.ServiceInstanceList
        roundtrip.go:199:       round tripping to sdkbroker.broker.k8s.io/v1alpha1, Kind=ServiceInstanceList v1alpha1.ServiceInstanceList
        roundtrip.go:199:       round tripping to sdkbroker.broker.k8s.io/v1alpha1, Kind=ServiceInstanceList v1alpha1.ServiceInstanceList
        roundtrip.go:199:       round tripping to sdkbroker.broker.k8s.io/v1alpha1, Kind=ServiceInstanceList v1alpha1.ServiceInstanceList
        roundtrip.go:199:       round tripping to sdkbroker.broker.k8s.io/v1alpha1, Kind=ServiceInstanceList v1alpha1.ServiceInstanceList
        roundtrip.go:199:       round tripping to sdkbroker.broker.k8s.io/v1alpha1, Kind=ServiceInstanceList v1alpha1.ServiceInstanceList
        roundtrip.go:199:       round tripping to sdkbroker.broker.k8s.io/v1alpha1, Kind=ServiceInstanceList v1alpha1.ServiceInstanceList
        roundtrip.go:199:       round tripping to sdkbroker.broker.k8s.io/v1alpha1, Kind=ServiceInstanceList v1alpha1.ServiceInstanceList
        roundtrip.go:199:       round tripping to sdkbroker.broker.k8s.io/v1alpha1, Kind=ServiceInstanceList v1alpha1.ServiceInstanceList
        roundtrip.go:199:       round tripping to sdkbroker.broker.k8s.io/v1alpha1, Kind=ServiceInstanceList v1alpha1.ServiceInstanceList
        roundtrip.go:199:       round tripping to sdkbroker.broker.k8s.io/v1alpha1, Kind=ServiceInstanceList v1alpha1.ServiceInstanceList
        roundtrip.go:199:       round tripping to sdkbroker.broker.k8s.io/v1alpha1, Kind=ServiceInstanceList v1alpha1.ServiceInstanceList
        roundtrip.go:199:       round tripping to sdkbroker.broker.k8s.io/v1alpha1, Kind=ServiceInstanceList v1alpha1.ServiceInstanceList
        roundtrip.go:199:       round tripping to sdkbroker.broker.k8s.io/v1alpha1, Kind=ServiceInstanceList v1alpha1.ServiceInstanceList
        roundtrip.go:199:       round tripping to sdkbroker.broker.k8s.io/v1alpha1, Kind=ServiceInstanceList v1alpha1.ServiceInstanceList
        roundtrip.go:199:       round tripping to sdkbroker.broker.k8s.io/v1alpha1, Kind=ServiceInstanceList v1alpha1.ServiceInstanceList
        roundtrip.go:199:       round tripping to sdkbroker.broker.k8s.io/v1alpha1, Kind=ServiceInstanceList v1alpha1.ServiceInstanceList
        roundtrip.go:199:       round tripping to sdkbroker.broker.k8s.io/v1alpha1, Kind=ServiceInstanceList v1alpha1.ServiceInstanceList
        roundtrip.go:199:       round tripping to sdkbroker.broker.k8s.io/v1alpha1, Kind=ServiceInstanceList v1alpha1.ServiceInstanceList
        roundtrip.go:141: round tripping sdkbroker.broker.k8s.io/__internal, Kind=ServiceInstance
        roundtrip.go:199:       round tripping to sdkbroker.broker.k8s.io/v1alpha1, Kind=ServiceInstance v1alpha1.ServiceInstance
panic: an unexported field was encountered, nested like this:  [recovered]
        panic: an unexported field was encountered, nested like this: int64 [recovered]
        panic: an unexported field was encountered, nested like this: time.Time -> int64 [recovered]
        panic: an unexported field was encountered, nested like this: unversioned.Time -> time.Time -> int64 [recovered]
        panic: an unexported field was encountered, nested like this: broker.ServiceInstanceCondition -> unversioned.Time -> time.Time -> int64 [recovered]
        panic: an unexported field was encountered, nested like this: []broker.ServiceInstanceCondition -> broker.ServiceInstanceCondition -> unversioned.Time -> time.Time -> int64 [recovered]
        panic: an unexported field was encountered, nested like this: broker.ServiceInstanceStatus -> []broker.ServiceInstanceCondition -> broker.ServiceInstanceCondition -> unversioned.Time -> time.Time -> int64 [recovered]
        panic: an unexported field was encountered, nested like this: broker.ServiceInstance -> broker.ServiceInstanceStatus -> []broker.ServiceInstanceCondition -> broker.ServiceInstanceCondition -> unversioned.Time -> time.Time -> int64 [recovered]
        panic: an unexported field was encountered, nested like this: *broker.ServiceInstance -> broker.ServiceInstance -> broker.ServiceInstanceStatus -> []broker.ServiceInstanceCondition -> broker.ServiceInstanceCondition -> unversioned.Time -> time.Time -> int64 [recovered]
        panic: an unexported field was encountered, nested like this: *broker.ServiceInstance -> broker.ServiceInstance -> broker.ServiceInstanceStatus -> []broker.ServiceInstanceCondition -> broker.ServiceInstanceCondition -> unversioned.Time -> time.Time -> int64

goroutine 11 [running]:
panic(0xe8a8c0, 0xc420394360)
        /usr/lib/golang/src/runtime/panic.go:500 +0x1a1
testing.tRunner.func1(0xc4200843c0)
        /usr/lib/golang/src/testing/testing.go:579 +0x25d
panic(0xe8a8c0, 0xc420394360)
        /usr/lib/golang/src/runtime/panic.go:458 +0x243
github.com/openshift/open-service-broker-sdk/vendor/k8s.io/apimachinery/third_party/forked/golang/reflect.makeUsefulPanic(0xf69ac0, 0xc4204f9900, 0x16)
        /home/pmorie/go/src/github.com/openshift/open-service-broker-sdk/vendor/k8s.io/apimachinery/third_party/forked/golang/reflect/deep_equal.go:96 +0xfc
panic(0xe8a8c0, 0xc420394340)
        /usr/lib/golang/src/runtime/panic.go:458 +0x243
github.com/openshift/open-service-broker-sdk/vendor/k8s.io/apimachinery/third_party/forked/golang/reflect.makeUsefulPanic(0xee7640, 0xc4204f9900, 0x199)
        /home/pmorie/go/src/github.com/openshift/open-service-broker-sdk/vendor/k8s.io/apimachinery/third_party/forked/golang/reflect/deep_equal.go:96 +0xfc
panic(0xe8a8c0, 0xc420394320)
        /usr/lib/golang/src/runtime/panic.go:458 +0x243
github.com/openshift/open-service-broker-sdk/vendor/k8s.io/apimachinery/third_party/forked/golang/reflect.makeUsefulPanic(0xe982c0, 0xc4204f9a10, 0x199)
        /home/pmorie/go/src/github.com/openshift/open-service-broker-sdk/vendor/k8s.io/apimachinery/third_party/forked/golang/reflect/deep_equal.go:96 +0xfc
panic(0xe8a8c0, 0xc4203942c0)
        /usr/lib/golang/src/runtime/panic.go:458 +0x243
github.com/openshift/open-service-broker-sdk/vendor/k8s.io/apimachinery/third_party/forked/golang/reflect.makeUsefulPanic(0xe00980, 0xc4204f9a10, 0x197)
        /home/pmorie/go/src/github.com/openshift/open-service-broker-sdk/vendor/k8s.io/apimachinery/third_party/forked/golang/reflect/deep_equal.go:96 +0xfc
panic(0xe8a8c0, 0xc4203942a0)
        /usr/lib/golang/src/runtime/panic.go:458 +0x243
github.com/openshift/open-service-broker-sdk/vendor/k8s.io/apimachinery/third_party/forked/golang/reflect.makeUsefulPanic(0xef5680, 0xc4202f6540, 0x199)
        /home/pmorie/go/src/github.com/openshift/open-service-broker-sdk/vendor/k8s.io/apimachinery/third_party/forked/golang/reflect/deep_equal.go:96 +0xfc
panic(0xe8a8c0, 0xc420394280)
        /usr/lib/golang/src/runtime/panic.go:458 +0x243
github.com/openshift/open-service-broker-sdk/vendor/k8s.io/apimachinery/third_party/forked/golang/reflect.makeUsefulPanic(0xf751e0, 0xc4202f6560, 0x199)
        /home/pmorie/go/src/github.com/openshift/open-service-broker-sdk/vendor/k8s.io/apimachinery/third_party/forked/golang/reflect/deep_equal.go:96 +0xfc
panic(0xe8a8c0, 0xc420394220)
        /usr/lib/golang/src/runtime/panic.go:458 +0x243
github.com/openshift/open-service-broker-sdk/vendor/k8s.io/apimachinery/third_party/forked/golang/reflect.makeUsefulPanic(0xf75500, 0xc4202f6560, 0x199)
        /home/pmorie/go/src/github.com/openshift/open-service-broker-sdk/vendor/k8s.io/apimachinery/third_party/forked/golang/reflect/deep_equal.go:96 +0xfc
panic(0xe8a8c0, 0xc4203941e0)
        /usr/lib/golang/src/runtime/panic.go:458 +0x243
github.com/openshift/open-service-broker-sdk/vendor/k8s.io/apimachinery/third_party/forked/golang/reflect.makeUsefulPanic(0xe11c00, 0xc4202f6560, 0x1a6)
        /home/pmorie/go/src/github.com/openshift/open-service-broker-sdk/vendor/k8s.io/apimachinery/third_party/forked/golang/reflect/deep_equal.go:96 +0xfc
panic(0xe8a8c0, 0xc4203941c0)
        /usr/lib/golang/src/runtime/panic.go:458 +0x243
github.com/openshift/open-service-broker-sdk/vendor/k8s.io/apimachinery/third_party/forked/golang/reflect.Equalities.deepValueEqual(0xc4204729c0, 0xe11c00, 0xc4202f6560, 0x1a6, 0xe11c00, 0xc4202f6620, 0x1a6, 0xc420290f20, 0x7, 0x390d9d9633157b00)
        /home/pmorie/go/src/github.com/openshift/open-service-broker-sdk/vendor/k8s.io/apimachinery/third_party/forked/golang/reflect/deep_equal.go:219 +0x590
github.com/openshift/open-service-broker-sdk/vendor/k8s.io/apimachinery/third_party/forked/golang/reflect.Equalities.deepValueEqual(0xc4204729c0, 0xf75500, 0xc4202f6560, 0x199, 0xf75500, 0xc4202f6620, 0x199, 0xc420290f20, 0x6, 0xfbc9246daa9f300)
        /home/pmorie/go/src/github.com/openshift/open-service-broker-sdk/vendor/k8s.io/apimachinery/third_party/forked/golang/reflect/deep_equal.go:186 +0xfab
github.com/openshift/open-service-broker-sdk/vendor/k8s.io/apimachinery/third_party/forked/golang/reflect.Equalities.deepValueEqual(0xc4204729c0, 0xf751e0, 0xc4202f6560, 0x199, 0xf751e0, 0xc4202f6620, 0x199, 0xc420290f20, 0x5, 0xc790a10e1b0ffd00)
```

**Comment by jmrodri** [comment]
Confirmed

**Comment by pmorie** [comment]
Closed by #7
